src/components/Main/Main.jsx

```diff
@@ -13,7 +13,8 @@ export default () => (
    <div className="App">
       <NavBar />
       <Switch>
-         <Route exact path="/" render={() => <LoginPage />} />
+         <Route exact path="/" render={() => <Dashboard />} />
+         {/* <Route exact path="/" render={() => <LoginPage />} /> */}
```

Comment:
Is there a reason you changed the routing?

Reply:
No was just testing something and forgot to change it back, my bad